.github/workflows/mobilecoin-workflow-dev-deploy.yaml

```diff
@@ -43,10 +43,17 @@ on:
       DEV_RANCHER_TOKEN:
         description: "Rancher access token"
         required: true
+      DEV_FOG_REPORT_B_SIGNING_CERT:
```

Comment:
"a" cert values are populated by the mobilecoin-workflow-dev-setup-environment.yaml action.  I'll make a second pass and see if I can make this more consistent.  For now I'm looking to minimize changes.